contracts/src/libraries/Errors.sol

```diff
@@ -0,0 +1,120 @@
+// SPDX-License-Identifier: Apache-2.0
+
+/*
+ * Copyright 2022, Specular contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+pragma solidity ^0.8.0;
+
```

Comment:
i think it might be more convenient to keep error definitions that are specific to certain functionality in the contract files they're meant for and use this file only for generic errors that are used across multiple contracts

Reply:
+1 @destroyersrt can you simply move these errors if only used by one component? thanks!